### DISCUSSION TITLE
What causes the custom context click to be obscured by the default behavior

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5691] [General]
![image](https://user-images.githubusercontent.com/9245110/197396659-4be657d0-28c2-4052-b1af-0c55cd2afb6c.png)

![image](https://user-images.githubusercontent.com/9245110/197396662-130bb3d1-59c5-4027-84dd-14908622cbb4.png)

I use HTTP agent and ssh forward to visist code-server-web. 

![image](https://user-images.githubusercontent.com/9245110/197396798-8485e32e-f333-44d8-8b4a-7e7171df4ec2.png)

The page displays fine, but when doing the right-click menu it also shows the default browser right-click menu, which I had to manually cancel to see the correct menu.

Obviously, this is not a direct problem with code-server, but is it a resource that is not loaded correctly that causes the problem?

**Comment by jsjoeio** [comment]
Hmm...I am not sure to be completely honest. have you tried asking [here](https://github.com/microsoft/vscode-discussions)?
